service-subscribe/package.json

```
@@ -1,19 +1,20 @@
 {
   "name": "subscribe-mailchimp",
-  "version": "1.0.1",
+  "version": "2.0.1",
```

why major? none of the routing changed, so it would be 
```suggestion
  "version": "1.1.0",
```